cmd/ipfs/init.go

```diff
@@ -24,7 +24,7 @@ import (
 
 const (
 	nBitsForKeypairDefault = 2048
```

Comment:
This const is now unused right?

Reply:
It's used in on more place: https://github.com/ipfs/go-ipfs/pull/7251/files#diff-54f2c4a5481de30b8ccb59ad5b6588b3R251
Thoughts?

Reply:
I don't think it's really worth keeping.

We can already do `ipfs init` without any bits flag and things still work since the size will default to the one used in config.CreateIdentity() for the RSA Type. Also, it'll end up actually getting used in zero places once we switch to Ed25519 keys by default since `ipfs daemon --init` will use the defaults, which won't include the RSA bits.